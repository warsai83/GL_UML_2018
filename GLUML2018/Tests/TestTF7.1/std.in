LOAD e.txt
DETAILSMALADIE M8
QUITTER