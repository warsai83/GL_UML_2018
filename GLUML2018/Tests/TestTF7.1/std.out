Bienvenue sur le service Malad'If ! 

 Saisir une action a effectuer : 

Succes de l'envoi du fichier d'empreintes
Chargement de BD...
Chargement termine
 Saisir une action a effectuer : 

Affichage des détails de la maladie ...
1;AA;18.713158;23.035148;173.707814;M8

 Saisir une action a effectuer : 

